app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    action, file, new_file = command.split(" ")
```

Comment:
you should check if command has 3 arguments before it and the first argument is correct